Framework/DataObjects/src/FractionalRebinning.cpp

```diff
@@ -87,11 +490,9 @@ void normaliseOutput(MatrixWorkspace_sptr outputWS,
     for (size_t j = 0; j < outputY.size(); ++j) {
       if (progress)
         progress->report("Calculating errors");
-      const double binWidth = outputX[j + 1] - outputX[j];
       double eValue = std::sqrt(outputE[j]);
-      // Don't do this for a RebinnedOutput workspace. The fractions
-      // take care of such things.
-      if (inputWS->isDistribution() && inputWS->id() != "RebinnedOutput") {
+      if (inputWS->isDistribution()) {
```

Comment:
If the workspace is a distribution _and_ and `RebinnedOutput` this check means that it would not do the dividing by binwidths but all the plotting algorithms afterwards still assumes this happens so the plotting gives numbers which are too big by the binwidth size.

I think the original thinking was that the `finalize()` routine (which divides by fractional area) takes care of this (that was the comment in the code), but it doesn't. The alternative is to switch off the distribution flag after `FractionalRebinning` so that the plotting routines does thing correctly.